src/config/src/config.rs

```diff
@@ -1398,6 +1398,12 @@ pub struct Limit {
         help = "maximum series to display in charts"
     )]
     pub max_dashboard_series: usize,
+    #[env_config(
+        name = "ZO_MINI_SEARCH_PARTITION_DURATION_SECS",
```

Comment:
Suggest rename to `ZO_SEARCH_MINI_PARTITION_xxx`, all the search env should have `ZO_SEARCH_` something like this.

Reply:
Got it, done!